flasc/floris_tools.py

```diff
@@ -11,302 +11,45 @@
-            # unc_pmfs=unc_pmfs,
-            # unc_options=unc_options
-        )
-
-        # Format the found solutions back to the dataframe format
-        Fp = interpolate.NearestNDInterpolator(
-            np.vstack([X.flatten(), Y.flatten()]).T,
-            np.reshape(turbine_powers, (-1, nturbs))
-        )
-        Fws = interpolate.NearestNDInterpolator(
-            np.vstack([X.flatten(), Y.flatten()]).T,
-            np.reshape(
-                np.mean(fi.floris.flow_field.u, axis=(3, 4)),
-                (-1, nturbs)
-            )
-        )
-        Fti = interpolate.NearestNDInterpolator(
-            np.vstack([X.flatten(), Y.flatten()]).T,
-            np.reshape(
-                fi.floris.flow_field.turbulence_intensity_field[:, :, :, 0, 0],
-                (-1, nturbs)
-            )
-        )
-
-        # Finally save solutions to the dataframe
-        df_out.loc[df_out.index, pow_cols] = Fp(df_out[["wd", "ws"]]) / 1000.0
-        df_out.loc[df_out.index, wd_cols] = np.tile(df_out["wd"], (nturbs, 1)).T
-        df_out.loc[df_out.index, ws_cols] = Fws(df_out[["wd", "ws"]])
-        df_out.loc[df_out.index, ti_cols] = Fti(df_out[["wd", "ws"]])
-
-    else:
-        # If cannot process in grid-style format, process one by one (SLOW)
-        for iii, idx in enumerate(df_out.index):
-            if (
-                verbose and
-                ((np.remainder(idx, 100) == 0) or idx == df_out.shape[0]-1)
-            ): 
-                print('  Progress: finished %.1f percent (%d/%d cases).'
-                    % (100.*idx/df_out.shape[0], idx, df_out.shape[0]))
-
-            # # Update model parameters, if present in dataframe
-            # if use_model_params:
-            #     params = df_out.loc[idx, 'model_params_dict']
-            #     fi.set_model_parameters(params=params, verbose=False)
-
-            fi.reinitialize(
-                wind_speeds=[df_out.loc[idx, 'ws']],
-                wind_directions=[df_out.loc[idx, 'wd']],
-                turbulence_intensity=df_out.loc[idx, 'ti']
-            )
-
-            fi.calculate_wake(np.expand_dims(yaw_rel[iii, :], axis=[0, 1]))
-            turbine_powers = np.squeeze(
-                fi.get_turbine_powers(
-                # include_unc=include_unc,
-                # unc_pmfs=unc_pmfs,
-                # unc_options=unc_options
-                )
-            )
-            df_out.loc[idx, pow_cols] = turbine_powers / 1000.
-            df_out.loc[idx, wd_cols] = np.repeat(
-                df_out.loc[idx, 'wd'],
-                nturbs   # Assumed to be uniform
-            ) 
-            df_out.loc[idx, ws_cols] = np.squeeze(
-                np.mean(fi.floris.flow_field.u, axis=(3, 4))
-            )
-            df_out.loc[idx, ti_cols] = np.squeeze(
-                fi.floris.flow_field.turbulence_intensity_field
-            )
-
-    return df_out
-
-
-def calc_floris(df, fi, num_workers, job_worker_ratio=5, include_unc=False,
```

Comment:
`calc_floris()` gets called by `calculate_wfpower_for_samples()` (from floris_sensitivity_analysis.py), but `calculate_wfpower_for_samples()` is only used in _legacy examples. Should we be considering `calculate_wfpower_for_samples()` redundant, or do we want to continue to support its use by leaving the `calc_floris()` (and, as a result, `_run_fi_serial()`) function available?

Reply:
I wrote those functions a long time ago and they're really outdated. I don't think they work anymore. I just wanted to keep them around in case we wanted to use them, to have a starting point. I'm also OK with removing them as a whole. We can always go back in the GitHub history to find them anyway.

Reply:
Sounds good. I'm also fine with removing them.